<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureViewer</class>
 <widget class="QMainWindow" name="TemperatureViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TemperatureViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Настройка</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="Connection">
         <item row="0" column="0">
          <widget class="QLabel" name="labelConnection">
           <property name="text">
            <string>Состояние подключения</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="statusConnection">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Неизвестно</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelCOM">
           <property name="text">
            <string>COM-порт</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cb_COM"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pb_ConnectDisconnect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Подключиться</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_Control">
         <property name="text">
          <string>Отправить контроль</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="Control">
         <item row="0" column="0">
          <widget class="QLabel" name="labelControl">
           <property name="text">
            <string>Результат контроля</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="statusControl">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Неизвестно</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
